uber/templates/registration/multi_merch_pickup.html

```diff
@@ -24,9 +46,11 @@
 
 <form method="post" action="">
 {% csrf_token %}
-This form is for when someone picks up badges for multiple other people.
+This form is for when someone picks up merch for multiple other people.
```

Comment:
This was an unfortunate typo, though I doubt it actually confused anyone in practice.  To be clear, we do NOT have a page where people pick up tons of badges for other people :)